tests/Message/AbstractResponseTest.php

```diff
@@ -1,77 +0,0 @@
-<?php
-
-namespace MyOnlineStore\Tests\Omnipay\KlarnaCheckout\Message;
-
-use MyOnlineStore\Omnipay\KlarnaCheckout\Message\AbstractResponse;
-use Omnipay\Tests\TestCase;
-
-class AbstractResponseTest extends TestCase
```

Comment:
Why has this test been removed? Looks like it is now covered by tests for **some** of it's implementations, but not all, which (imo) is an unclear practice.